flow/declarations/File.js

```diff
@@ -2,3 +2,4 @@ declare class File {
   name: string,
   path: string,
 }
+export default File;
```

Comment:
@daniloprates why do we need this export here?

Reply:
It was returning a lint error. But couldn't reproduce, so removed in the latest commit.